src/renderers/shaders/ShaderChunk/lights_pars.glsl

```diff
@@ -63,19 +63,9 @@ vec3 getAmbientLightIrradiance( const in vec3 ambientLightColor ) {
 
 		float lightDistance = length( lVector );
 
-		if ( testLightInRange( lightDistance, pointLight.distance ) ) {
-
-			directLight.color = pointLight.color;
-			directLight.color *= punctualLightIntensityToIrradianceFactor( lightDistance, pointLight.distance, pointLight.decay );
-
-			directLight.visible = true;
-
-		} else {
-
-			directLight.color = vec3( 0.0 );
-			directLight.visible = false;
-
-		}
+		directLight.color = pointLight.color;
+		directLight.color *= punctualLightIntensityToIrradianceFactor( lightDistance, pointLight.distance, pointLight.decay );
+		directLight.visible = ( directLight.color != vec3( 0.0, 0.0, 0.0 ) );
```

Comment:
The issue with COLOR_BLACK is that color can be RGB or RGBA, so we would need to defines RGB_BLACK and RGBA_BLACK but really it is only saving a few characters.  I did replace my usage of vec3( 0.0, 0.0, 0.0 ) with vec3( 0.0 ) which is what I should have done in the first place.